package-lock.json

```diff
@@ -16,7 +16,7 @@
         "@bcrs-shared-components/confirm-dialog": "1.2.1",
         "@bcrs-shared-components/contact-info": "1.2.4",
         "@bcrs-shared-components/corp-type-module": "1.0.9",
-        "@bcrs-shared-components/court-order-poa": "2.1.4",
+        "@bcrs-shared-components/court-order-poa": "^3.0.0",
```

Comment:
I think you need to re-run `npm i` = this should not have the caret character, because we don't want to auto-update this package, even to a patch version. We want to update imports manually, and have a look at the changes.

Reply:
Done, thanks @severinbeauvais